lib/app/custom_widgets/live_match_tile/elements/live_match_tile_logo.dart

```diff
@@ -12,7 +13,7 @@ class LiveMatchTileLogo extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     const String stringPlaceholder = '';
-    final String assetName = liveMatch.league?.flag ??
+    final String flag = liveMatch.league?.flag ??
```

Comment:
Consider using static values for default urls etc, like in `AppColors`

Reply:
going to do that in separate file exactly like AppColors